acceptance/README.md

```diff
@@ -1,257 +1,467 @@
-
-### Preserving Hosts
-
-If you need to ssh into the hosts after a test run, you can use the following sequence:
-
-    bundle exec rake ci:test_and_preserve_hosts CONFIG=some/config.yaml SHA=12345 TEST=a/foo_test.rb
-
-to get the initial templates provisioned, and a local log/latest/preserve_config.yaml created for them.
-
-Then you can log into the hosts, or rerun tests against them by:
-
-    bundle exec rake ci:test_against_preserved_hosts TEST=a/foo_test.rb
-
-This will use the existing hosts.
-
-NOTE: If you want configuration information to be preserved for all runs (potentially allowing you to run ci:test_against_preserved_hosts for any previous run that failed, and who's hosts were preserved, regardless of whether you initiated with a ci:test_and_preserve_hosts call) then you should add a ':__preserve_config__ => true' to your local_options.rb.
-
-### Cleaning Up Preserved Hosts
-
-If you run a number of jobs with --preserve_hosts or vi ci:test_and_preserve_hosts, you may eventually generate a large number of stale vms.  They should be reaped automatically by qa infrastructure within a day or so, but you may also run:
-
-    bundle exec rake ci:release_hosts
-
-to clean them up sooner and free resources.
-
-There also may be scenarios where you want to specify the host(s) to release. E.g. you may want to release a subset of the hosts you've created. Or, if a test run terminates early, ci:release_hosts may not be able to derive the name of the vm to delete. In such cases you can specify host(s) to be deleted using the HOST_NAMES environment variable. E.g.
-
-    HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
-    HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
+Always clean up after yourself
+```
+bundle exec beaker destroy
+```
 
-### Syncing local development code with VMPooler Windows machines
+-------------
 
-If you want to copy local code to a Windows machine that is running in beaker, you can use the `ci:sync:windows` rake task:
+## Configuration
+### Environment Variables
+A detailed description of the available environment variables can be found by
+running `bundle exec rake ci:help`. This will print a list of both required and
+optional environment variable with short descriptions on how they are used.
+Please review all of these options as they will impact how your test servers
+are provisioned. This rake task is the most up to date source for this
+information.
+
+### Customizing Test Targets
+If you are using the vmpooler hypervisor and the internal pooling api, you can
+customize the platforms to test on using the HOSTS environment variable. The
+internal pooling api is only available to Puppet, Inc. employees. You'll set the
+`HOSTS` environment variable to the host string you want to test, such as
+`HOSTS=redhat7-64ma-windows2012r2-64a`.
+
+If you are unsure of the syntax you need, you can verify the host string against
```

Comment:
I'd change this to be more clear we're talking about platform-specific syntax. Maybe something along the lines of, "For a list of available `HOSTS` platforms and their exact naming structures, check the keys listed in [beaker hostgenerator](). Generally, ..."